tests/data/placebo/test_glacier_untag/tagging.UntagResources_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
+            "HTTPStatusCode": 200, 
+            "RequestId": "488bc109-36ad-11e7-bfbf-47f7816c1ac2", 
+            "HTTPHeaders": {
+                "x-amzn-requestid": "488bc109-36ad-11e7-bfbf-47f7816c1ac2", 
+                "date": "Fri, 12 May 2017 00:52:33 GMT", 
+                "content-length": "25", 
+                "content-type": "application/x-amz-json-1.1"
+            }
+        }, 
+        "FailedResourcesMap": {}
```

Comment:
note this notion of inline failure in the response instead of an exception raised on error, we need to explicitly check this and log.